cmd/livepeer/livepeer.go

```diff
@@ -390,6 +390,12 @@ func main() {
 
 			n.Recipient.Start()
 			defer n.Recipient.Stop()
+
+			n.Balances = core.NewBalances(smCleanupInterval)
+			// Run n.Accounts.Cleanup() Routine
+			go n.Balances.Cleanup()
+			// Stop the cleanup routine on program exit
+			defer n.Balances.Stop()
```

Comment:
Should be `StopCleanup()`

Reply:
Yup, This is addressed in the commit that includes the rebase